config/locales/account_reset/en.yml

```diff
@@ -21,6 +21,9 @@ en:
         phone number.
     delete_account:
       are_you_sure: Are you sure you want to delete your account?
+      cancelled: Your request to delete your account was canceled. If you’d like to
```

Comment:
This is true. This PR is for a different scenario--when a user chooses that they want to delete, then receive the e-mail, then decide that they should log back in again. After the user continues with their chosen authentication method, they will see the banner that the deletion request was cancelled.

Reply:
Yep, I'm more looking at this part:

>If you’d like to delete your account, please submit the request again and complete the instructions. Do not log back into your account.

If we want to suggest the path for them to delete their account, we shouldn't suggest the 24 hour waiting period account deletion when they can click "Delete Account" and have it done immediately.

Reply:
> If we want to suggest the path for them to delete their account, we shouldn't suggest the 24 hour waiting period account deletion when they can click "Delete Account" and have it done immediately.

Ah, I see how this can be confusing. We are going to go back and make this path clearer. Thanks for pointing this out.